crates/sail-execution/src/driver/actor/handler.rs

```diff
@@ -122,18 +122,12 @@ impl DriverActor {
             warn!("worker {worker_id} not found");
             return ActorAction::Continue;
         };
-        let timeout = self.options().worker_launch_timeout;
-        if let WorkerState::Pending { pending_at } = &worker.state {
-            if pending_at.elapsed() > timeout {
```

Comment:
Some probe events do not need the time instant inside the event since they will not be updated (e.g. `pending_at`). But other probe events do need the time instance since they can be updated (e.g. `updated_at`).